tests/installation/system_role/validate_default_role.pm

```diff
@@ -25,10 +25,10 @@ use testapi;
 use Test::Assert 'assert_equals';
 
 sub run {
-    my $default    = get_test_suite_data()->{system_role}{default};
-    my $controller = $testapi::distri->get_system_role_controller();
+    my $default     = get_test_suite_data()->{system_role}{default};
```

Comment:
the original idea was to replace also this module (although I don't think that is reflected directly in the ticket), otherwise, we still have test data spread in each test, so there is no gain. I mean there is gain in the sense that we now can just use 'select' without test data, but the default in this case we cannot say that we will do later and put in some extended test suite, because the role selection depends on the product modules installed previously.

Could you apply the same for default role ( we could call it /system_role/default_role_text_mode.pm) and so on, and remove completely test data in each test suite. Otherwise, please file a ticket for it.

Reply:
I can do this, even though it was not indeed mentioned in the ticket at all, I assumed it might be covered by another one.

Reply:
we also don't need to check more than once for the same role in the same group this default.